jax/_src/lax/windowed_reductions.py

```diff
@@ -80,23 +90,68 @@ def reduce_window(operand, init_value, computation: Callable,
     return monoid_reducer(operand, window_dimensions, window_strides, padding,
                           base_dilation, window_dilation)
   else:
-    flat_init_avals = map(lax._abstractify, flat_init_values)
-    jaxpr, out_tree = lax._variadic_reduction_jaxpr(
-        computation, tuple(flat_init_avals), init_value_tree)
+    if is_jvp:
+      # TODO(mvoz): Check robustness
+      init_value_primals = init_value[0]
+      init_value_tangents = init_value[1]
```

Comment:
```suggestion
      init_value_primals, init_value_tangents = init_value
```

Does this work? If so it gives us an assertion for free (and saves a line).